storage/client/queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -124,6 +127,9 @@ public URL getQueueUrl() {
      *
      * {@codesnippet com.azure.storage.queue.queueAsyncClient.create#map}
      *
+     * <p>For more information, see the
```

Comment:
Adding Rest docs link in javadocs, 
Is this some standard guideline that all track 2 SDKs need to follow ?
Also, what happens when REST API docs link updates to a new version which doesn't correspond to our implemented SDK?

Reply:
This isn't a guideline for Track 2 SDKs, more of a nice to have feature given the depth of complexity in some of the Storage REST APIs. Additionally, Blobs has REST API doc links in previous versions and in Preview 1, so it is being consistent with it.

Reply:
@g2vinay in general, changes to our rest surface are only additive, and each parameter that was an addition at some point in time has a minimum version prominent in the description. If there is ever a breaking change to our rest surface, it is handled on a case by case basis. However, we don't diverge documentation pages. I would be impressed if one of these links became obsolete or problematic.

Reply:
Huh. So I'm looking into our links a bit more. It seems that we may break links, but I'm having trouble finding a scenario where we would direct you to the wrong one.